extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -359,4 +394,25 @@ describes.realWin('amp-ad-exit', {
         .to.have.been.calledWith(
         'http://localhost:8000/tracking?bar=bar', '');
   });
+
+  it('should replace custom URL variables with 3P Analytics signals', () => {
+    const open = sandbox.stub(win, 'open', () => {
+      return {name: 'fakeWin'};
+    });
+
+    addToResponseMap(env.ampdoc, TEST_3P_VENDOR, env.win.document.baseURI, {
```

Comment:
Done, in new test: it('should replace custom URL variables with 3P Analytics defaults')